### ISSUE TITLE
Failed to connect SQLite database with SQLTools

### ISSUE THREAD

**Issue by juneX05** [mtxr/vscode-sqltools#307]
Every time I connect my sqlite database it prompts
 "You need to install "sqlite3@4.0.8" to connect to database.sqlite."
And after I finish installing it asks to connect to the database then it repeats the prompt again

I wonder why it fails to connect with SQLite

**Desktop (please complete the following information):**
 - SQLTools Version [e.g. v0.19.7]
 - VSCode Version: [e.g 1.36.1]
 - Dialect Used/Version: [SQLite]
 - OS: [Windows]

**Comment by MLutt** [comment]
Same here, at least you're not alone. On Mint19/Ubuntu18.04/Linux via the snap store, just installed SQLTools...

**Comment by mtxr** [comment]
Can you send me the logs?

**Comment by MLutt** [comment]
Unfortunately, the logs don't seem to provide anything of use. However, here you go - zipped the whole logs directory of a fresh instance after trying to connect to the sqlite DB, getting the prompt to install sqlite3@4.0.8 and trying to open the connection
[20190730T205617.zip](https://github.com/mtxr/vscode-sqltools/files/3448455/20190730T205617.zip)

**Comment by MLutt** [comment]
> "sqltools.useNodeRuntime": true

My settings json has been cleared - reinstating the setting (@juneX05 https://vscode-sqltools.mteixeira.dev/connections/sqlite ) got it to work again...

However, now I'm getting syntax errors upon opening the connection so that much to that^^ - That's nothing to do with the issue though.

@mtxr I'd say feel free to close

**Comment by mtxr** [comment]
Ok. I'll close. 

But can you describe this syntax errors a bit more? Is it something caused by the extension? I didn't get it

**Comment by OzzyTheGiant** [comment]
Not to put pressure but this driver should ideally be updated to work with the latest Node LTS version, as I haven't been able to get any connections working for one particular project. Then I switched to Ubuntu and I can't do connections anymore on ANY project, constantly getting the "install sqlite3@4.2.0" prompt even after I install. Tried on Node 12, 14, and 15, using nodeRuntime setting and nothing works.
